packages/frama-c-base/frama-c-base.20150201/opam

```diff
@@ -87,4 +88,8 @@ conflicts: [
   "lablgtk" { < "2.18.2" } #for ocaml >= 4.02.1
 ]
 
-available: [ ocaml-version >= "3.12" & ocaml-version != "4.02.0" ]
+available: [
+  ocaml-version >= "3.12"
```

Comment:
Thanks for this! It looks like 3.12 isnt supported though:

```
#=== ERROR while installing frama-c-base.20151002 =============================#
# opam-version 1.2.2
# os           linux
# command      ./configure --prefix /home/travis/.opam/3.12.1 --disable-gui
# path         /home/travis/.opam/3.12.1/build/frama-c-base.20151002
# compiler     3.12.1
# exit-code    1
# env-file     /home/travis/.opam/3.12.1/build/frama-c-base.20151002/frama-c-base-28066-736596.env
# stdout-file  /home/travis/.opam/3.12.1/build/frama-c-base.20151002/frama-c-base-28066-736596.out
# stderr-file  /home/travis/.opam/3.12.1/build/frama-c-base.20151002/frama-c-base-28066-736596.err
### stdout ###
# [...]
# configure: ******************
# configure: * CONFIGURE MAKE *
# configure: ******************
# checking for make... make
# checking version of make... 3.81
# configure: *****************************
# configure: * CONFIGURE OCAML COMPILERS *
# configure: *****************************
# checking for ocamlc... ocamlc
# checking version of OCaml... 3.12.1
### stderr ###
# configure: error: Incompatible OCaml version; use 4.0+ (except 4.02.2, 4.02.0 and 4.00.0).
```

So `>= 4.0` should do the trick.